sdcm/cluster.py

```diff
@@ -2868,54 +2876,54 @@ def verify_logging_from_nodes(nodes_list):
         for node in nodes_list:
             if not os.path.exists(node.database_log):
                 error_msg = "No db log from node [%s] " % node
-                logger.warning(error_msg)
+                LOGGER.warning(error_msg)
```

Comment:
remove this line, we raise exception with error message

Reply:
That's right, but in some cases the exception might be suppressed cause other code. raise exception doesn't me we'll have traces of it in the logs